.nuspec/proguard.cfg

```diff
@@ -4,4 +4,7 @@
 -keep class android.support.multidex.MultiDexApplication { *; }
 -keep class android.support.design.internal.BaselineLayout { *; }
 -dontwarn android.support.design.internal.BaselineLayout
--keep class com.google.firebase.provider.FirebaseInitProvider { *; }
\ No newline at end of file
+-keep class com.google.firebase.provider.FirebaseInitProvider { *; }
+-keep class androidx.appcompat.widget.AlertDialogLayout { *; }
```

Comment:
@mattleibow @Redth 
Leaving these here for now ok?

Reply:
This should be good. If there are rules that don't match anything. The worst I think that can happen is that it will be a warning - but I don't think that reaches the user...